neutron/assetlist.json

```diff
@@ -368,6 +368,35 @@
         }
       ]
     },
+    {
+      "description": "Laika the Space Dog",
+      "denom_units": [
+        {
+          "denom": "neutron1yh2lv497ex0e7j62ewyx6x08uxcnvjlcvlusyr73v4azctyysursee7lh7",
+          "exponent": 0,
+          "aliases": [
+            "ulaika"
+          ]
+        },
+        {
+          "denom": "laika",
+          "exponent": 6
+        }
+      ],
+      "base": "neutron1yh2lv497ex0e7j62ewyx6x08uxcnvjlcvlusyr73v4azctyysursee7lh7",
+      "name": "LAIKA",
+      "display": "laika",
+      "symbol": "LAIKA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/laika.png"
+      },
+      "coingecko_id": "laika",
```

Comment:
This CoinGecko ID doesn't describe this asset, but a different ethereum asset. Please remove this coingecko id